### PR TITLE
Update user-agent string

### PR DESCRIPTION
**Pull request by anshumansworld**
This include to provide an option to add custom user-agent string in the request instead of default one i.e. Dalvik

### CODE REVIEW

**Comment by dvmrmc** [comment]
👍  good, thanks, but the version bump is achieved directly in the master branch

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This repository contains a tool to navigate deep into an URL redirection and ope
 
 Add the following line to your build.gradle file
 
-`compile "net.pubnative:url_driller:1.2.1"`
+`compile "net.pubnative:url_driller:1.3.0"`
```
Comment (dvmrmc) [inline README.md]: This doesn't work like this, please remove this change

File: url_driller/build.gradle [hunk 2]
```diff
@@ -13,7 +13,7 @@ android {
     defaultConfig {
         minSdkVersion 10
         targetSdkVersion 21
-        versionCode 4
+        versionCode 5
         versionName sharedVersion
```
Comment (dvmrmc) [inline url_driller/build.gradle:17]: Also all build.gradle changes need to be reverted

File: README.md (hunk 1 above)
Comment (anshumansworld) [inline README.md]: 👍

File: url_driller/build.gradle (hunk 2 above)
Comment (anshumansworld) [inline url_driller/build.gradle:17]: 👍

**Pull request merged**